### PR TITLE
feat(docs): add link to new deploy apps tutorial

### PR DESCRIPTION
**Pull request by robin-norwood**
This adds a link to the new [CDKTF applications](https://learn.hashicorp.com/tutorials/terraform/cdktf-applications?in=terraform/cdktf) tutorial.

### CODE REVIEW

File: website/docs/cdktf/concepts/stacks.html.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ description: "Use stacks to specify separate collections of infrastructure for d
 
 -> **Note:** CDK for Terraform is currently in [beta](/docs/cdktf/index.html#project-maturity-and-production-readiness).
 
+> **Hands-on:** Try the [Deploy Applications with CDK for Terraform](https://learn.hashicorp.com/tutorials/terraform/cdktf-applications?in=terraform/cdktf&utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorial on HashiCorp Learn.
```
Comment (laurapacilio) [inline website/docs/cdktf/concepts/stacks.html.md:12]: I love this - thank you for adding! I wonder - can we move it to below the description of what stacks is? Just right above the single stack header?

Also! I just realized that the headings seem to be off here - could you change all three of these on this page to be H2s?

**Review by laurapacilio — CHANGES_REQUESTED** [review]
Thank you! Some quick suggested changes :-)

File: website/docs/cdktf/concepts/stacks.html.md (hunk 1 above)
Comment (ansgarm) [reply website/docs/cdktf/concepts/stacks.html.md]: Fixed them @laurapacilio, please confirm :)

File: website/docs/cdktf/concepts/stacks.html.md [hunk 2]
```diff
@@ -108,17 +110,17 @@ All Terraform operations are currently limited to a single stack, so you must sp
 
 To run multiple stacks at once, move them into the same directory and run `cdktf deploy`. For example, you could create a combined directory called `multiple-stacks-dev`, run `cdktf deploy multiple-stacks-dev`, and all Terraform operations will run in the folder `cdktf.out/stacks/multiple-stacks-dev`.
 
-##### Cross Stack References
+### Cross Stack References
```
Comment (laurapacilio) [inline website/docs/cdktf/concepts/stacks.html.md:113]:
```suggestion
### Cross-Stack References
```

File: website/docs/cdktf/concepts/stacks.html.md [hunk 3]
```diff
@@ -108,17 +110,17 @@ All Terraform operations are currently limited to a single stack, so you must sp
...
 
 CDKTF does not yet support referencing resources from another stack automatically. You can achieve this manually with [outputs](/docs/cdktf/concepts/variables-and-outputs.html) and the [remote state data source](/docs/language/state/remote-state-data.html).
 
 Please refer to this [issue](https://github.com/hashicorp/terraform-cdk/issues/651) if you are interested in using cross stack references.
```
Comment (laurapacilio) [inline website/docs/cdktf/concepts/stacks.html.md:117]:
```suggestion
Please refer to this [issue](https://github.com/hashicorp/terraform-cdk/issues/651) if you are interested in using cross-stack references.
```

**Review by laurapacilio — APPROVED** [review]
Two nits from me and then you can merge :-) Thank you!!

**Pull request merged**